### DISCUSSION TITLE
Prefill title / content

### DISCUSSION THREAD

**Discussion by nfroidure** [community/community#2892] [Discussions]
Would be nice to be able to prefill the title like with Stack Overflow :
https://stackoverflow.com/questions/ask?title=My%20Title%20Prefilled&tags=testing&tags=code&body=xxxx

I would allow me to suggest users to post their question on a given error code in GH Discussions instead of Stack Overflow : https://github.com/nfroidure/whook/blob/52401fb11615153230dd654b800aa730ff11c94a/packages/whook-http-router/src/errorHandler.ts#L50

**Comment by styfle** [comment]
Also it would be nice to pre-fill category.

This used to work with the `<REPO>/discussions/new?category_id=66161` query string param but it stopped working.

**Reply by BetaHuhn** [reply]
Already answered in [this discussion](https://github.com/github/feedback/discussions/2999#discussioncomment-250805)

You have to use the title of the category to pre-fill it:

- https://github.com/github/feedback/discussions/new?category=discussions-feedback
- https://github.com/github/feedback/discussions/new?category=mobile-feedback

> **Note:** If your category contains white space or an ampersand, use a dash (`-`) instead.
